settings.gradle.kts

```
@@ -132,6 +132,8 @@ hideFromDependabot(":smoke-tests:images:servlet:servlet-3.0")
 hideFromDependabot(":smoke-tests:images:servlet:servlet-5.0")
 hideFromDependabot(":smoke-tests:images:spring-boot")
 
+include(":smoke-tests-otel-starter")
```

Do we want dependabot to update e.g. h2 or dbcp2 versions?
```suggestion
hideFromDependabot(":smoke-tests-otel-starter")
```

Also, WDYT about using e.g.
```suggestion
include(":smoke-tests:spring-starter-smoke-test")
```
? I'm trying to avoid exploding the number of directories in the root repo dir, we could put all smoke tests projects into `smoke-test`